apis/vpcresources/v1alpha1/cninode_types.go

```diff
@@ -0,0 +1,57 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// FeatureName is a type of feature name supported by EKS networking. It can be Security Group for Pods, custom networking, or others
+type FeatureName string
+
+const (
+	SecurityGroupsForPods FeatureName = "SecurityGroupsForPods"
+	CustomNetworking      FeatureName = "CustomNetworking"
+)
+
+// Important: Run "make" to regenerate code after modifying this file
+// CNINodeSpec defines the desired state of CNINode
+type CNINodeSpec struct {
+	Features []FeatureName `json:"features,omitempty"`
+}
+
+type VPCResource string
```

Comment:
```suggestion
```

Reply:
Missed this when remove the block of const. Will update.